src/runtime/experiment-flags.js

```diff
@@ -60,4 +60,9 @@ export const ExperimentFlags = {
    * Experiment flag for guarding changes to fix PayClient redirect flow.
    */
   PAY_CLIENT_REDIRECT: 'pay-client-redirect',
+
+  /**
+   * Enables scrolling within dialogs
```

Comment:
nit: missing period at the end of this comment. 

it doesn't seem to be consistently applied in this file, but I've always used punctuation, capitalization, etc, in my JSDoc comments.

Reply:
Added!